script/add_record.py

```diff
@@ -0,0 +1,25 @@
+import json
+from datetime import datetime
+
+JSON_FILE = 'init/persons-test.json'
+persons = json.load(open(JSON_FILE, encoding='utf-8'))
+
+
+def add_record_to_json():
```

Comment:
Function should only do one thing. In this case just add new record (not save data)
Also it should get all necessary data as a parameters